### ISSUE TITLE
Add tests for CSS validation

### ISSUE THREAD

**Issue by fulldecent** [fulldecent/html-website-template#19]
# Discussion

There is probably already a tool for this we can use.

**Comment by ilyalyo** [comment]
If we will use npm, we can use this tool: [CSSLint](https://github.com/CSSLint/csslint)
Also, [purifycss](https://github.com/purifycss/purifycss) is a very nice tool, may be you would like to use it too, it analyze all css and html/php files and check which selectors are used, than it can generate 1 optimized css file.

**Comment by fulldecent** [comment]
Not interested in purify because that is a build tool, we only want validation tools now. Let's use CSSLint. Have you checked for a Ruby wrapper?

**Comment by ilyalyo** [comment]
I found one wrapper, but it's not maintained more than 5 years, I think it's not a problem becouse we can run it with sh like this:
```ruby 
desc "Validate css files under css folder"
task :css_lint do
  unless system('which csslint')
   puts "csslint is not installed\nrun npm install -g csslint"
   exit(1)
  end
  sh "csslint #{@build_dir}css/*.css"
end
```

**Comment by fulldecent** [comment]
Let's wait until we find a more robust solution.

> run npm install -g csslint

is too manual. I really do not want to add install steps for the end user.

**Comment by ilyalyo** [comment]
What about this one: https://github.com/w3c-validators/w3c_validators
It can validate css and it's pretty active, last commit was last month.

**Comment by fulldecent** [comment]
Yes, that looks good!

**Comment by ilyalyo** [comment]
Just a note: In this library we can validate either string (css) or css file, we can not validate css inside html file without extracting it.
I think we can add this as htmlproofer extension, what do you think?

**Comment by fulldecent** [comment]
That is an interesting idea, but not in scope for now.

**Comment by ilyalyo** [comment]
You want to implement directly in lighting sites?
I think it easier with htmlproofer.
Or it should be stand alone gem.

**Comment by fulldecent** [comment]
Right now we just want to validate .css files using an off-the-shelf library. We do not want to validate css inside html files.

**Comment by ilyalyo** [comment]
Added PR: https://github.com/fulldecent/lightning-sites/pull/20

**Comment by fulldecent** [comment]
Fixed in #20 and released in fa757fe1d225a2629bd46fca1b96e8047fe3975a
